Dialog/RoomConfigDialog.cpp

```diff
@@ -218,12 +252,24 @@ void RoomConfigDialog::on_ComboBox_TilesetID_currentIndexChanged(int index)
         }
         else
         {
-            elements << QString::number(WL4Constants::BGLayerDisableDefaultPtr, 16).toUpper();
+            elements << QString::number(WL4Constants::BGLayerDefaultPtr, 16).toUpper();
         }
         ui->ComboBox_BGLayerPicker->addItems(elements);
 
+        ui->CheckBox_BGLayerEnable->setChecked(true);
+        ui->CheckBox_Layer0Enable->setChecked(false); ui->CheckBox_Layer0Enable->setChecked(true);
+        ui->CheckBox_Layer2Enable->setChecked(true);
+        if(ui->ComboBox_TilesetID->currentIndex() == 0x21)
```

Comment:
Please add a comment describing the significance of "0x21" or make it a constant in WL4Constants

Reply:
0x21 is also used on line 181

Reply:
done